### ISSUE TITLE
torch version

### ISSUE THREAD

**Issue by MrAcademic** [rdevon/DIM#25]
For Resnet Do You think it works on pytorch version 1.2.0?
Could you please test it?
 python scripts/main.py local classifier --d.source ImageFolder --encoder_config resnet19_32x32 --local.mode nce --local.mi_units 1024 -n DIM_Resnet --t.epochs 1000

which version of pytorch do you recommend or tested?

**Comment by rdevon** [comment]
I am currently using 1.1, but I can test 1.2 sometime this week. Did you try this and get an error?

**Comment by MrAcademic** [comment]
Actually not for all other commands (like training CFAR or comparisons to CPC. For STL-10 on folded 64x64 Alexnet ) BUT for that specific command I get:
Data setup--------------------------------------------------------------------------------------------------------
Visualization setup-----------------------------------------------------------------------------------------------
Model setup-------------------------------------------------------------------------------------------------------
/Users/distiller/project/conda/conda-bld/pytorch_1565272526878/work/aten/src/ATen/native/LegacyDefinitions.cpp:14: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/Users/distiller/project/conda/conda-bld/pytorch_1565272526878/work/aten/src/ATen/native/LegacyDefinitions.cpp:14: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
Optimizer setup---------------------------------------------------------------------------------------------------
Killed: 9
!

**Comment by jingcx** [comment]
pytorch 1.2 works well using command line.
